### PR TITLE
fix(city guide): fix cta images not showing up

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Bugfix**

This PR resolves [CX-693]

### Description

After @admbtlr's suggestion, upgrading RN to the latest fixed the image issue 🙏.

I'd like to test the tipsi-stripe thing, maybe we can do this in QA? I saw that in the Podfile.lock it used to point to a specific commit, but now it's not. Of course, as always, after an upgrade, we should do our normal QA anyway.

Before
cityGuideiOS14

After
Screenshot 2020-11-09 at 11 49 26

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

#known 

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-693]: https://artsyproduct.atlassian.net/browse/CX-693

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -68,56 +75,6 @@ target 'Artsy' do
 
   pod 'Emission', path: './emission'
 
-  pod 'React', path: 'node_modules/react-native'
-  if ENV['CIRCLE_BUILD_NUM']
-    pod 'React-Core', path: 'node_modules/react-native'
-  else
-    pod 'React-Core', path: 'node_modules/react-native', subspecs: ['DevSupport']
```
Comment (ashfurrow) [inline Podfile:75]: We should make sure that the beta builds don't ship with this enabled 😅

**Comment by pvinis** [comment]
This should be ready now 👍.

**Review by ashfurrow — APPROVED** [review]
Looks good! Just one note about testing the next beta. I think we can merge when ready 👍

File: Podfile (hunk 1 above)
Comment (pvinis) [reply Podfile:75]: mm I'll check this, one sec.

Comment (pvinis) [reply Podfile:75]: I'm seeing this: https://github.com/facebook/react-native/blob/d03c0f965ed26c7c8a5557ab183396b64d25a9a1/scripts/react_native_pods.rb#L14-L15

Comment (ashfurrow) [reply Podfile:75]: Okay, so it seems we can move this to when we invoke `use_react_native!`?

```ruby
use_react_native! production:  ENV['CIRCLE_BUILD_NUM']
```

Comment (pvinis) [reply Podfile:75]: Yep! Took me a while, cause node_modules broke again 😅. Let's see if that makes ci happy.

**Comment by pvinis** [comment]
This should be good to go.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Developer Specific file shouldn't be changed, you can skip by adding #known to the PR body and re-running CI</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by pvinis** [comment]
Oh no.. this had the squashongreen and I rebased it on top of code 12 to make it green 😬. #4117 is the one. Since it's green and approved too, I'll do a tiny change and we are good.

**Pull request merged**